js-envs/src/main/scala/org/scalajs/jsenv/phantomjs/PhantomJSEnv.scala

```diff
@@ -115,7 +115,7 @@ class PhantomJSEnv(
     private[this] val recvBuf = mutable.Queue.empty[String]
     private[this] val fragmentsBuf = new StringBuilder
 
-    private def comSetup = {
+    override protected def comSetup(): Seq[VirtualJSFile] = {
```

Comment:
Hum ... This should not have been made protected. It also does not make sense that this method be defined as high up as `ComJSRunner`.